src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericDateInput.js

```diff
@@ -20,17 +20,21 @@ export const GenericDateInput = ({ parameterData, context, parameterValue, setPa
     maxDateMessage: t('genericcomponent.dateInput.error.maxDateMessage', 'Maximum date is not respected'),
     invalidDateMessage: t('genericcomponent.dateInput.error.invalidDateMessage', 'Date is invalid'),
   };
-
+  const errorMessages = {
+    required: t('views.scenario.scenarioParametersValidationErrors.required', 'This field is required'),
+  };
   return (
     <BasicDateInput
       key={parameterData.id}
       id={parameterData.id}
       label={t(`solution.parameters.${parameterData.id}`, parameterData.id)}
       tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterData.id), '')}
-      value={parameterValue ?? new Date()}
+      value={parameterValue ?? {}}
```

Comment:
In `BasicDateInput` component, we call methods that are specific to Date objects such as `value.toLocaleDateString()` (when edition is disabled). Will it work if we send an empty object `{}` instead of `new Date()`?
Maybe we should allow null values (and make the prop not `required`) instead of using an empty object.

Reply:
Good idea, thanks ! If we make a prop not `required`, we need to do the same in the ui package?